<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>MainWindow</class>
<widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
        <rect>
            <x>0</x>
            <y>0</y>
            <width>1200</width>
            <height>800</height>
        </rect>
    </property>
    <property name="windowTitle">
        <string>照片海报拼接软件</string>
    </property>
    
    <!-- 中央部件 -->
    <widget class="QWidget" name="centralWidget">
        <layout class="QHBoxLayout" name="mainLayout">
            <item>
                <!-- 左侧控制面板 -->
                <widget class="QWidget" name="leftPanel">
                    <property name="minimumWidth">
                        <number>200</number>
                    </property>
                    <property name="maximumWidth">
                        <number>300</number>
                    </property>
                    <layout class="QVBoxLayout" name="leftLayout">
                        <!-- 模板选择区域 -->
                        <item>
                            <widget class="QGroupBox" name="groupBoxTemplate">
                                <property name="title">
                                    <string>海报模板</string>
                                </property>
                                <layout class="QVBoxLayout" name="verticalLayout">
                                    <item>
                                        <widget class="QListWidget" name="templateList">
                                            <property name="selectionMode">
                                                <enum>QAbstractItemView::SingleSelection</enum>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnCustomTemplate">
                                            <property name="text">
                                                <string>自定义模板</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        
                        <!-- 图片管理区域 -->
                        <item>
                            <widget class="QGroupBox" name="groupBoxPhotos">
                                <property name="title">
                                    <string>图片管理</string>
                                </property>
                                <layout class="QVBoxLayout" name="verticalLayout_2">
                                    <item>
                                        <widget class="QPushButton" name="btnCamera">
                                            <property name="text">
                                                <string>打开摄像头</string>
                                            </property>
                                            <property name="icon">
                                                <iconset resource="">
                                                    <normaloff>camera_icon.png</normaloff>
                                                </iconset>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnCapture">
                                            <property name="text">
                                                <string>拍照</string>
                                            </property>
                                            <property name="enabled">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnLoad">
                                            <property name="text">
                                                <string>导入图片</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QScrollArea" name="scrollArea">
                                            <property name="widgetResizable">
                                                <bool>true</bool>
                                            </property>
                                            <widget class="QWidget" name="scrollAreaWidgetContents">
                                                <property name="geometry">
                                                    <rect>
                                                        <x>0</x>
                                                        <y>0</y>
                                                        <width>180</width>
                                                        <height>200</height>
                                                    </rect>
                                                </property>
                                                <layout class="QVBoxLayout" name="photosLayout">
                                                    <property name="spacing">
                                                        <number>5</number>
                                                    </property>
                                                </layout>
                                            </widget>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        
                        <!-- 编辑工具区域 -->
                        <item>
                            <widget class="QGroupBox" name="groupBoxTools">
                                <property name="title">
                                    <string>编辑工具</string>
                                </property>
                                <layout class="QGridLayout" name="gridLayout">
                                    <item row="0" column="0">
                                        <widget class="QPushButton" name="btnCrop">
                                            <property name="text">
                                                <string>裁剪</string>
                                            </property>
                                            <property name="enabled">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="1">
                                        <widget class="QPushButton" name="btnRotate">
                                            <property name="text">
                                                <string>旋转</string>
                                            </property>
                                            <property name="enabled">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="0">
                                        <widget class="QPushButton" name="btnFilter">
                                            <property name="text">
                                                <string>滤镜</string>
                                            </property>
                                            <property name="enabled">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="1">
                                        <widget class="QPushButton" name="btnAdjust">
                                            <property name="text">
                                                <string>调整</string>
                                            </property>
                                            <property name="enabled">
                                                <bool>false</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="2" column="0" colspan="2">
                                        <widget class="QSlider" name="zoomSlider">
                                            <property name="minimum">
                                                <number>10</number>
                                            </property>
                                            <property name="maximum">
                                                <number>200</number>
                                            </property>
                                            <property name="value">
                                                <number>100</number>
                                            </property>
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        
                        <!-- 输出区域 -->
                        <item>
                            <widget class="QGroupBox" name="groupBoxOutput">
                                <property name="title">
                                    <string>输出</string>
                                </property>
                                <layout class="QVBoxLayout" name="verticalLayout_3">
                                    <item>
                                        <widget class="QPushButton" name="btnSave">
                                            <property name="text">
                                                <string>保存海报</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnExport">
                                            <property name="text">
                                                <string>导出高清</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnClear">
                                            <property name="text">
                                                <string>清空重做</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QComboBox" name="comboFormat">
                                            <item>
                                                <property name="text">
                                                    <string>PNG格式</string>
                                                </property>
                                            </item>
                                            <item>
                                                <property name="text">
                                                    <string>JPEG格式</string>
                                                </property>
                                            </item>
                                            <item>
                                                <property name="text">
                                                    <string>PDF格式</string>
                                                </property>
                                            </item>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        
                        <!-- 拉伸空白 -->
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            
            <!-- 右侧工作区 -->
            <item>
                <widget class="QSplitter" name="splitter">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <!-- 预览区域 -->
                    <widget class="QWidget" name="previewWidget">
                        <layout class="QVBoxLayout" name="previewLayout">
                            <item>
                                <widget class="QGraphicsView" name="graphicsView">
                                    <property name="dragMode">
                                        <enum>QGraphicsView::RubberBandDrag</enum>
                                    </property>
                                    <property name="renderHints">
                                        <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QStatusBar" name="statusBar">
                                    <property name="sizeGripEnabled">
                                        <bool>false</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    
                    <!-- 摄像头区域 -->
                    <widget class="QWidget" name="cameraWidget">
                        <layout class="QVBoxLayout" name="cameraLayout">
                            <item>
                                <widget class="QLabel" name="labelCamera">
                                    <property name="text">
                                        <string>摄像头</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QWidget" name="cameraContainer">
                                    <layout class="QVBoxLayout" name="cameraContainerLayout"/>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="btnCloseCamera">
                                    <property name="text">
                                        <string>关闭摄像头</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    
    <!-- 菜单栏 -->
    <widget class="QMenuBar" name="menuBar">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1200</width>
                <height>25</height>
            </rect>
        </property>
        <widget class="QMenu" name="menuFile">
            <property name="title">
                <string>文件</string>
            </property>
            <addaction name="actionNew"/>
            <addaction name="actionOpen"/>
            <addaction name="actionSave"/>
            <addaction name="separator"/>
            <addaction name="actionExport"/>
            <addaction name="separator"/>
            <addaction name="actionExit"/>
        </widget>
        <widget class="QMenu" name="menuEdit">
            <property name="title">
                <string>编辑</string>
            </property>
            <addaction name="actionUndo"/>
            <addaction name="actionRedo"/>
            <addaction name="separator"/>
            <addaction name="actionCrop"/>
            <addaction name="actionRotate"/>
            <addaction name="actionFilter"/>
        </widget>
        <widget class="QMenu" name="menuView">
            <property name="title">
                <string>视图</string>
            </property>
            <addaction name="actionZoomIn"/>
            <addaction name="actionZoomOut"/>
            <addaction name="actionFitView"/>
        </widget>
        <widget class="QMenu" name="menuHelp">
            <property name="title">
                <string>帮助</string>
            </property>
            <addaction name="actionAbout"/>
        </widget>
        <addaction name="menuFile"/>
        <addaction name="menuEdit"/>
        <addaction name="menuView"/>
        <addaction name="menuHelp"/>
    </widget>
    
    <!-- 工具栏 -->
    <widget class="QToolBar" name="mainToolBar">
        <attribute name="toolBarArea">
            <enum>TopToolBarArea</enum>
        </attribute>
        <attribute name="toolBarBreak">
            <bool>false</bool>
        </attribute>
        <addaction name="actionNew"/>
        <addaction name="actionOpen"/>
        <addaction name="actionSave"/>
        <addaction name="separator"/>
        <addaction name="actionCrop"/>
        <addaction name="actionRotate"/>
        <addaction name="separator"/>
        <addaction name="actionZoomIn"/>
        <addaction name="actionZoomOut"/>
    </widget>
    
    <!-- 动作定义 -->
    <action name="actionNew">
        <property name="text">
            <string>新建</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+N</string>
        </property>
    </action>
    <action name="actionOpen">
        <property name="text">
            <string>打开</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+O</string>
        </property>
    </action>
    <action name="actionSave">
        <property name="text">
            <string>保存</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+S</string>
        </property>
    </action>
    <action name="actionExport">
        <property name="text">
            <string>导出...</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+E</string>
        </property>
    </action>
    <action name="actionExit">
        <property name="text">
            <string>退出</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+Q</string>
        </property>
    </action>
    <action name="actionUndo">
        <property name="text">
            <string>撤销</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+Z</string>
        </property>
    </action>
    <action name="actionRedo">
        <property name="text">
            <string>重做</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+Y</string>
        </property>
    </action>
    <action name="actionCrop">
        <property name="text">
            <string>裁剪</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+R</string>
        </property>
    </action>
    <action name="actionRotate">
        <property name="text">
            <string>旋转</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+T</string>
        </property>
    </action>
    <action name="actionFilter">
        <property name="text">
            <string>滤镜</string>
        </property>
    </action>
    <action name="actionZoomIn">
        <property name="text">
            <string>放大</string>
        </property>
        <property name="shortcut">
            <string>Ctrl++</string>
        </property>
    </action>
    <action name="actionZoomOut">
        <property name="text">
            <string>缩小</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+-</string>
        </property>
    </action>
    <action name="actionFitView">
        <property name="text">
            <string>适合窗口</string>
        </property>
        <property name="shortcut">
            <string>Ctrl+0</string>
        </property>
    </action>
    <action name="actionAbout">
        <property name="text">
            <string>关于</string>
        </property>
    </action>
</widget>
<resources/>
<connections/>
</ui>
